addons/mail/static/src/models/user_setting/user_setting.js

```diff
@@ -130,23 +129,18 @@ registerModel({
          * @param {number} param0.volume
          */
         async saveVolumeSetting({ guestId, partnerId, volume }) {
-            this._debounce(async () => {
-                await this.async(() => this.env.services.rpc(
-                    {
-                        model: 'res.users.settings',
-                        method: 'set_volume_setting',
-                        args: [
-                            [this.messaging.currentUser.resUsersSettingsId],
-                            partnerId,
-                            volume,
-                        ],
-                        kwargs: {
-                            guest_id: guestId,
-                        },
-                    },
-                    { shadow: true },
-                ));
-            }, 5000, `sound_${partnerId}`);
+            if (this.timeouts[`sound_${partnerId}`]) {
+                browser.clearTimeout(this.timeouts[`sound_${partnerId}`]);
+            }
+            this.update({
+                timeouts: {
+                    ...this.timeouts,
+                    [`sound_${partnerId}`]: browser.setTimeout(
```

Comment:
I would still much prefer if we cleaned this up before merging to not have object in field.